sysobject.ids

```diff
@@ -2720,6 +2720,8 @@
 45.3.78.2	Avaya	NETWORKING	Ethernet Routing Switch 4850GTS-PWR
 45.3.78.4	Avaya	NETWORKING	ERS 4850GTS
 45.3.79.1	Avaya	NETWORKING	VSP-7024XLS
+45.3.80.2	Nortel	NETWORKING	UNKNOWN
+45.3.80.4	Nortel	NETWORKING	UNKNOWN
```

Comment:
Why set model to `UNKNOWN` here ? Do you mean that sysobjectid can be used by different models ?
What happen for the device if you set nothing here ?

Reply:
I've just put what the tool gave me, since I don't know the expected format.

Reply:
One idea of the sysObjectID file and particularly thanks to the fourth field is to provide a model name when it is known and when only one model is related to one sysObjectID.
When not know or when many models share the same sysObjectID, the agent still can find the model by another way and then we don't fill that field. So forcing it to `UNKNOWN` is just wrong if you just don't know the model as this may not permit the agent to search for the real model name.

So can you answer that question: do you know if many models share that sysObjectID values ?

Reply:
`So can you answer that question: do you know if many models share that sysObjectID values ?`
Yes I can answer, no I don't if many models share the same values. Scan has been done in a huge infrastructure composed of hundreds of network devices.

Reply:
Than I can suggest you to only target a partial sysObjectID match so if `45.3.80` is the base of the sysObjectID, the manufacturer should be set to `Nortel`.
```suggestion
45.3.80	Nortel	NETWORKING
```